### ISSUE TITLE
Support REUSE Specification

### ISSUE THREAD

**Issue by cmeessen** [fair-software/howfairis#362]
We have been applying the [REUSE Specification](https://reuse.software) to the [Research Software Directory](https://github.com/research-software-directory/RSD-as-a-service) to indicate copyright and individual licenses on a per-file basis. Unfortunately, howfairis does not detect licenses when they are specified according to the REUSE specification and fails.

We were wondering whether it is planned to add support for the REUSE specification?

Details about the specification can be found on their webpage under [Specification](https://reuse.software/spec/).

**Comment by juhannc** [comment]
> I believe the best approach is to check the `REUSE.toml` file for support.

Unfortunately, you cannot just check for the `REUSE.toml` file, as it is optional according to the REUSE Spec:

> Licensing Information MAY be associated with a file through a REUSE.toml file

A project can be fully licensed according to the REUSE spec with just the `LICENSES/` folder and all license information in the file headers. Thus, you should at least also check for the `LICENSES/` folder and possibly also the `DEP5` file. Still, this only indicates, that a project _tries_ to be REUSE compliant, not that it actually is.

<https://reuse.software/spec-3.2/>

**Comment by cmeessen** [comment]
Great to see that this issue is being picked up.

I suggest to check the error code of `reuse lint` when running it in the root directory of the repository. If the error code is `0`, the repository is reuse compliant.

**Comment by juhannc** [comment]
Yes, that would obviously be the best solution but also require integrating `reuse` in this project. Calling `reuse lint` via a subprocess is not a very nice solution, it's not very pythonic and could create new security issues. Instead I think a minimal reimplementation of their cli would probably be the best solution

**Comment by fdiblen** [comment]
@cmeessen @juhannc 
FYI, I started to implement the reuse support in #377. It uses https://api.reuse.software/ to check the compliance. So, the repository needs to be registered to be considered compliant.

**Comment by fdiblen** [comment]
ping @jmaassen @ewan-escience

**Comment by fdiblen** [comment]
done in #377
